Lib/test/test_posix.py

```diff
@@ -1660,14 +1660,17 @@ def test_setsigdef_wrong_type(self):
 
     @requires_sched
     def test_setscheduler_only_param(self):
+        if sys.platform.startswith(('freebsd', 'netbsd', 'openbsd', 'gnukfreebsd')):
+            raise unittest.SkipTest("test may fail on BSD")
```

Comment:
We already skip a similar test on FreeBSD and NetBSD:

```
        # POSIX states that calling sched_setparam() or sched_setscheduler() on
        # a process with a scheduling policy other than SCHED_FIFO or SCHED_RR
        # is implementation-defined: NetBSD and FreeBSD can return EINVAL.
        if not sys.platform.startswith(('freebsd', 'netbsd')):
```

See commit b402a5c01b2010fc2409e5cb137cde86c4b6cbf3.

Please use the decorator, skipIf or skipUnless:
    @unittest.skipUnless(sys.platform == 'win32',
                         'test specific to the Windows console')

Please also mention the bpo number in the skip message: "bpo-34685: " prefix.